### DISCUSSION TITLE
Native Dark mode introduced in Zotero 7.0.0-beta.55!

### DISCUSSION THREAD

**Discussion by toineenzo** [tefkah/zotero-night#175] [General]
Since the latest beta 55 for Zotero 7, they finally introduced native dark mode functionality!
Therefore this plugin is not needed anymore and will also make the latest beta look a little bit weird (colorwise).
I want to thank @tefkah for his hard work :)

**Comment by tefkah** [comment]
oh wow, that's great!

thank you for your kind words :)

i see that they're completely revamped the UI, making the plugin not work anymore with the latest version of the beta.

with that i'll probably stop developing this plugin. I had bigger ambitions for it to be some kind of theme manager, but never really got around to it, and am kind of glad I didn't in retrospect: what the zotero team has done is wayyy better than i've ever could've managed!
as i think there are other plugins that allow you to customize the colors of the pdf, i don't think there's a big need for this one anymore (unless ofc you want to use if for zotero 6)

big thanks to the zotero team and everyone who supported me until now, now onto bigger things :)

**Comment by peytonstafford** [comment]
This sounds wonderful, but I am on version 6.0.37. How do I update to 7?
